examples/python/config.py.in

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+##===-----------------------------------------------------------------------------*- Python -*-===##
```

Comment:
python-bock comments are usually just one `#` and not two (https://www.python.org/dev/peps/pep-0008/#block-comments)

Reply:
I moved everything (python) to PEP8 except the makers, since they would break API and rupert is using it at the moment. 
I created another PR for doing this as well